Description: Fix menu-item drag url
 When dragging a menu item to the desktop, it isn't correctly copied because
 of an incorrect patch. This fixes that.
Author: Tsu Jan <tsujan2000@gmail.com>
Bug: https://github.com/lxqt/lxqt/issues/1512
Applied-Upstream: commit:a26e700
Last-Update: 2018-07-13
--- a/qtxdg/xdgmenuwidget.cpp
+++ b/qtxdg/xdgmenuwidget.cpp
@@ -170,7 +170,7 @@ void XdgMenuWidgetPrivate::mouseMoveEven
         return;
 
     QList<QUrl> urls;
-    urls << QUrl(a->desktopFile().fileName());
+    urls << QUrl::fromLocalFile(a->desktopFile().fileName());
 
     QMimeData *mimeData = new QMimeData();
     mimeData->setUrls(urls);
